src/ihex_parse.c

```diff
@@ -60,7 +65,7 @@ ihex_recordset_t* ihex_rs_from_file(const char* filename)
 	
 	ihex_recordset_t* r;
 	
-	fd = open(filename, O_RDONLY);
+	fd = open(filename, O_RDONLY | O_BINARY);
```

Comment:
This does not compile when `O_BINARY` is defined as empty:

>ihex_parse.c:68:41: error: expected expression before ‘)’ token

See https://travis-ci.org/martin-helmich/libcintelhex/jobs/71804401#L401
Defining `O_BINARY` as `0` might be safer.